packages/types/environment.d.ts

```diff
@@ -31,6 +31,8 @@ declare namespace NodeJS {
     readonly TANDEM_CLIENT_ID: string | undefined;
     readonly TANDEM_CLIENT_SECRET: string | undefined;
     readonly TANDEM_BASE_URL: string | undefined;
+    readonly WEBEX_CLIENT_ID: string | undefined;
```

Comment:
We need to find a way to stop polluting the global env directly. I think every app should expose it's environment and if needed we can import that env here.

Reply:
not making any changes here

Reply:
Yeah that was to create an issue out of it for future work.